cylc/flow/broadcast_mgr.py

```diff
@@ -70,12 +70,10 @@ def check_ext_triggers(self, itasks, ext_trigger_queue):
             ext_trigger = ext_trigger_queue.get_nowait()
             self.ext_triggers.setdefault(ext_trigger, 0)
             self.ext_triggers[ext_trigger] += 1
-        return set(
-            [
-                itask for itask in itasks
-                if self._match_ext_trigger(itask)
-            ]
-        )
+        return {
+            itask for itask in itasks
+            if self._match_ext_trigger(itask)
+        }
```

Comment:
Saves building and destroying an intermediate list.

Reply:
It seems I wasn't aware of the existence of set comphrehensions (until a recent previous review, anyway) .. :facepalm: